src/features/wallet/root/WalletView.tsx

```diff
@@ -46,10 +76,15 @@ const WalletView = ({ layout, animationPoints }: Props) => {
   }, [navigation])
 
   const animateActivityToBottom = useCallback(() => {
```

Comment:
maybe we can rename this `toggleShowReceive` or something since that's what it's doing in both states?

Reply:
Good point, done